Formula/tfel.rb

```diff
@@ -17,16 +17,24 @@ class Tfel < Formula
 
   depends_on "cmake" => :build
   depends_on "gcc" => :build
-  depends_on "python@3.10" => :build
+  depends_on "boost-python3"
+  depends_on "python@3.10"
   fails_with gcc: "5"
 
+  patch do
+    url "https://github.com/thelfer/tfel/raw/fca68680750847fd56315664ecad19089bf9f0b7/patches/homebrew/tfel-4.0.0.patch"
```

Comment:
Is this actually accepted upstream? Or is it just hosted there for Homebrew?

Reply:
Changes are being pushed into the development branch

Reply:
@SMillerDev Is the patch will also be applied to the master branch with the `--head` option?

Reply:
In the current form, yes. You can wrap it in a `stable` block when it's merged

Reply:
Patch wrapped and modification merged here : https://github.com/thelfer/tfel/commit/f610e638fde512cc1d94127458c89297e0ddaab3